docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md

```diff
@@ -266,6 +280,126 @@ az acr connected-registry show \
 
 You may need to a wait few minutes until the deployment of the connected registry completes.
 
+## Deploy the api proxy module on IoT Edge
+
+Add api proxy module from Azure Marketplace `IoT Edge API Proxy`
+
+Add the following two environment variables in addition to the existing ones:
+
+```
+"CONNECTED_ACR_ROUTE_ADDRESS": {
+      "value": "connected-registry:8080"
```

Comment:
Maybe it will be good to link to the API proxy setup documentation for reference.

Reply:
Done.